### DISCUSSION TITLE
Uploading the Code - Photo Doesn't Quite Match My Platformio Folder View

### DISCUSSION THREAD

**Discussion by daveclifford** [DCC-EX/EX-T3-WiFi#78] [General]
ref: https://github.com/DCC-EX/EX-T3-WiFi/blob/master/docs/code.md
![platformio-firebeetle2-ksnip](https://user-images.githubusercontent.com/1976894/227364384-3806ff55-fe01-4be8-b8d7-71ef8e2d3a13.png)

Note:  My PlatformIO (Linux V1.76.2) has a FireBeetle2 Folder/Subfolder structure and the tasks are within that, not as shown in the **Uploading the Code** .md file.

Maybe a nit [https://www.merriam-webster.com/dictionary/nit - second defition], but this may help others find their way.......

(Sorry the blue highlighted line is not on the Build Filesystem Image command line)

Dave

**Comment by alex-code** [comment]
Hi Dave,
Is it just the position that I'd clipped the image at not showing the firebeetle2 folder?

**Comment by daveclifford** [comment]
Look at PROJECT TASKS/firebeetle2/General and Plafrom tasks. 

Your screenshot clips don't show the firebeetle2 folder.  
Some interested hobbyists will compare screens and wonder why theirs is different than yours .........

It is no big deal to me, but, to some, maybe, it may cause them to ask .......

Dave
